### DISCUSSION TITLE
State of Nextcloud App

### DISCUSSION THREAD

**Discussion by ReasonableRaptor** [cloud-py-api/nc_py_api#160] [Q&A] [answered]
Hi! 
Thank you for this awesome project! I was about to install it on my NC server, but one question still remains: 
I'd like to use nc_py_api, but the nextcloud app store points to [cloud_py_api](https://github.com/cloud-py-api/cloud_py_api). Therefore my question: 

Are you planning to update the cloud_py_api NC app, or will nc_py_api be packaged as an installable nextcloud app at some point? 

I hope I understood everything correctly and my question makes sense - I am new to all this nextcloud-app-store stuff and am not entirely sure how everything is interlinked :-)

**Comment by bigcat88** [comment, marked as answer]
cloud_py_api has been replaced by [AppAPI](https://github.com/cloud-py-api/app_api) , which currently is in active development state.

This particular repository is a Python library for working with Nextcloud. It will never be packaged as an application(as it does not need to), but it can be part of any application written for AppAPI.

**Reply by ReasonableRaptor** [reply]
Ah, okay, thanks for the clarification - that helped me a lot! 

And thanks for your awesome work! I'll give nc_py_api a try in the coming days!
